cmd/query/app/querysvc/adjuster/standard.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package adjuster
+
+import (
+	"time"
+)
+
+// StandardAdjusters returns a list of adjusters applied by the query service
+// before returning the data to the API clients.
+func StandardAdjusters(maxClockSkewAdjust time.Duration) []Adjuster {
+	return []Adjuster{
+		SpanIDUniquifier(),
+		SortAttributesAndEvents(),
+		SpanHash(),                    // requires SortAttributesAndEvents for deterministic results
+		ClockSkew(maxClockSkewAdjust), // adds warnings (which affect SpanHash) on duplicate span IDs
+		IPAttribute(),
+		ResourceAttributes(),
+		SpanLinks(),
```

Comment:
We should consider renaming these adjusters to reflect the business function. Reading this list I can reasonably guess what the first two are doing, plus ClockSkew. But all others are poorly named.

My suggestion would be to use verbs that describe the function, e.g.
  * DedupeClientServerSpanIDs
  * SortAttributes
  * DedupeSpans
  * etc

Reply:
@yurishkuro Done. Let me know what you think